spec.html

```diff
@@ -34306,17 +34306,17 @@ <h1>String.prototype.slice ( _start_, _end_ )</h1>
       </emu-clause>
 
       <emu-clause id="sec-string.prototype.split">
-        <h1>String.prototype.split ( _separator_, _limit_ )</h1>
-        <p>This method returns an Array into which substrings of the result of converting this object to a String have been stored. The substrings are determined by searching from left to right for occurrences of _separator_; these occurrences are not part of any String in the returned array, but serve to divide up the String value. The value of _separator_ may be a String of any length or it may be an object, such as a RegExp, that has a @@split method.</p>
+        <h1>String.prototype.split ( _separator_, _substringLimit_ )</h1>
+        <p>This method finds substrings separated by _separator_ in its *this* value and returns them in an Array, truncated as necessary to satisfy _substringLimit_ (in which case later contents are absent rather than collected into an element that includes _separator_). Behaviour is completely delegated to the @@split method of _separator_ if present (e.g., because _separator_ is a RegExp instance). Otherwise, a _substringLimit_ coerced to zero produces an empty Array, an *undefined* _separator_ produces a single-element Array containing only the string-coerced *this* value, any _separator_ coerced to the empty String splits the *this* value into individual code units, and all other normal completion results have at least one element (which will be the entire string-coerced *this* value if it _separator_ is not present).</p>
```

Comment:
```suggestion
        <p>This method finds substrings separated by _separator_ in its *this* value and returns them in an Array, truncated as necessary to satisfy _substringLimit_ (in which case later contents are absent rather than collected into an element that includes _separator_). Behaviour is completely delegated to the @@split method of _separator_ if present (e.g., because _separator_ is a RegExp instance). Otherwise, a _substringLimit_ coerced to zero produces an empty Array, an *undefined* _separator_ produces a single-element Array containing only the string-coerced *this* value, any _separator_ coerced to the empty String splits the *this* value into individual code units, and all other normal completion results have at least one element (which will be the entire string-coerced *this* value if _separator_ is not present).</p>
```